### ISSUE TITLE
Feature Request: Easy Delete

### ISSUE THREAD

**Issue by djringjr** [k9mail/k-9#2798]
Please search to check for an existing issue (including closed issues, for which the fix may not have yet been released) before opening a new issue: https://github.com/k9mail/k-9/issues?q=is%3Aissue

### Expected behavior
Tell us what should happen

### Actual behavior
Tell us what happens instead

### Steps to reproduce
1.
2.
3.

### Environment
K-9 Mail version:

Android version:

Account type (IMAP, POP3, WebDAV/Exchange):

**Comment by djringjr** [comment]
Please make it possible to delete messages easier.

**Comment by philipwhiuk** [comment]
There's delete actions:

* When long clicking on a message.
* When the message is selected
* In the action bar when a message is being viewed.
* In a notification as a quick action.

The only thing we are potentially missing is an on swipe action and that is probably part of the material redesign. I'm going to close this because there's no specific request.

**Comment by djringjr** [comment]
Hi Phil,

The time to press down on the message is very long.  I get a lot of doubtful messages - not worth reading - advertisements, offers, groups that I barely follow.  I don't worry about the mail I do want to read it is the  mail i want to get rid of that I want to get rid of quickly.  Sorry you closed this, ,it would be grand to have some swipe action to throw the messages away.  My favorite email program is mutt, you  can get rid of lots of messages and scan them quickly, K-9 is my favorite for android except for the deleting.  It is way too tedious.  Often I don't open up a message  before I delete it.

Thanks,

Daviid

**Comment by philipwhiuk** [comment]
You can enable checkboxes so you don't have to 'press down long' to select messages.
